project/SparkBuild.scala

```diff
@@ -476,7 +476,8 @@ object SparkParallelTestGrouping {
     "org.apache.spark.ml.classification.LinearSVCSuite",
     "org.apache.spark.sql.SQLQueryTestSuite",
     "org.apache.spark.sql.hive.thriftserver.ThriftServerQueryTestSuite",
-    "org.apache.spark.sql.hive.thriftserver.SparkSQLEnvSuite"
+    "org.apache.spark.sql.hive.thriftserver.SparkSQLEnvSuite",
+    "org.apache.spark.sql.hive.thriftserver.ui.ThriftServerPageSuite"
```

Comment:
This fixes the test failure against Hive 2.3 (tested in https://github.com/apache/spark/pull/26706); however, I have no explicit evidence. Just given my speculation and it worked.

Reply:
I'm still seeing failures like in https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/4953/testReport/ - should we make all the thriftserver tests single-threaded?

Reply:
Ah yeah. Maybe that's a better idea. I'll monitor a bit more and make a PR soon